lib/Dialect/FIRRTL/Transforms/LowerIntrinsics.cpp

```diff
@@ -53,14 +66,14 @@ LogicalResult LowerIntrinsicsPass::initialize(MLIRContext *context) {
 
 // This is the main entrypoint for the lowering pass.
 void LowerIntrinsicsPass::runOnOperation() {
-  auto result = lowering->lower(getOperation());
-  if (failed(result))
-    return signalPassFailure();
-
-  numConverted += *result;
 
-  if (*result == 0)
-    markAllAnalysesPreserved();
+  TypeSwitch<Operation *>(&(*getOperation()))
+      .Case<FModuleOp, FExtModuleOp, FIntModuleOp, FMemModuleOp, ClassOp,
```

Comment:
The copy-and-pasting of this block is increasing the ops being visited by a lot of passes.  Very few passes deal with FExtModuleOp, for example, as they don't have bodies.